src/exampleTypes/ResultTable.js

```diff
@@ -86,14 +86,15 @@ export default InjectTreeNode(
             {_.map(
               x => (
                 <tr key={x._id}>
-                  {_.map(
-                    ({ field, display = x => x, Cell = 'td' }) => (
-                      <Cell key={field}>
-                        {display(getRecord(x)[field], getRecord(x))}
-                      </Cell>
-                    ),
-                    schema
-                  )}
+                  {_.map(({ field, display, Cell = 'td' }) => {
+                    let record = getRecord(x)
+                    let props = { key: field }
+                    // React warns you if you send record as a property to `td`
+                    if (Cell !== 'td') props.record = record
```

Comment:
What's the reasoning on this? The display function is passed the record, why would you want it on an overridable component as well? Display can return a react component, so for conditional formatting you'd do it in there - the only reason to actually override the cell component is if a `td` doesn't work for some reason.

Reply:
The idea is to use `display` 99% of the time since in theory that can be reused by multiple components.